index.html

```diff
@@ -431,13 +430,18 @@
         immediately after the wake lock is released.
       </div>
       <p>
-        A <a>browsing context</a> is <dfn>requesting the wake lock</dfn> of
-        type <var>type</var> if and only if the following procedure returns
+        A <a>Document</a> is <dfn>requesting the wake lock</dfn> of type
+        <var>type</var> if and only if the following procedure returns
         <code>true</code>:
       </p>
       <ol>
-        <li>Let <var>navigator</var> be the <a>Navigator</a> object associated
-        with this <a>browsing context</a>.
+        <li>Let <var>document</var> be this <a>Document</a> object.
```

Comment:
"this Document" is still ambiguous. Maybe say: 

> Let document be the current settings object's responsible document.

(with appropriate cross references)

Similarly,
> Let `<var>navigator</var> be the `<var>document</var>` default view's Navigator object instance.

And window is just the client settings object, I believe (or just the document's default view).